File: cap03/exemplo09-boxplot.r
# Carregamento da Bibilioteca
library(ggplot2)

# Utilizando o dataset nativo Iris
dados = iris

# Plotando um boxplot por espécies da característica Petal.length
ggplot(dados, aes(x=Species, y=Petal.Length, fill=Species)) + geom_boxplot()

File: cap03/exemplo09-boxplot.py
# Imports das Bibiliotecas
import seaborn as sns
import matplotlib.pyplot as plt

# Carregando o dataset
dados = sns.load_dataset('iris')

# Plotando um boxplot por especies da característica petal length 
sns.boxplot(x=dados["species"], y=dados["petal_length"])

plt.show()